providers/catalog.go

```diff
@@ -66,6 +66,14 @@ const (
 	Zuora                               Provider = "zuora"
 )
 
+func optionalString(s string) *string {
```

Comment:
Instead of this, can we use `x-go-type-skip-optional-pointer`? ([ref](https://github.com/deepmap/oapi-codegen?tab=readme-ov-file#openapi-extensions))

Reply:
oh neat! did not know about that.